packages/web/src/common/store/player/sagas.ts

```diff
@@ -75,7 +75,7 @@ export function* watchPlay() {
   const isOfflineModeEnabled = yield* call(
     getFeatureEnabled,
     FeatureFlags.OFFLINE_MODE_ENABLED
-  ) ?? false
+  ) ?? true
```

Comment:
Do we still want this?

Reply:
Yes, unfortunately we don't have context in common to do this properly. However, it shouldn't be possible to get here unless you've already managed to download and trigger a play for an offline track so... I think that's fine?